### PR TITLE
Rename LISK -> LSK for Lisk

### PR DESCRIPTION
**Pull request by kirillshevch**
Fix based on issue [Naming consistency "Lisk"](https://github.com/kirillshevch/react-cryptocoins/issues/9)

### CODE REVIEW

**Comment by kirillshevch** [comment]
@allienworks :point_up_2:

File: webfont/cryptocoins.css [hunk 1]
```diff
@@ -505,11 +505,11 @@
   content: "\E079";
 }
 
-.cc.LISK-alt::before {
+.cc.LSK-alt::before {
```
Comment (allienworks) [inline webfont/cryptocoins.css]: Both of these changes are not correct. This file gets generated automatically by Gulp. First, the SVG icon itself needs to be renamed to `LSK.svg`, then you'd run `gulp` inside of project's folder. That would recompile all the font's CSS and mainly the webfonts themselves.

Comment (kirillshevch) [reply webfont/cryptocoins.css]: thanks, fixed

**Comment by allienworks** [comment]
Thanks very much for your contribution!

**Pull request merged**